exercises/circular-buffer/src/example.h

```diff
@@ -0,0 +1,30 @@
+#ifndef CIRCULAR_BUFFER_H
+#define CIRCULAR_BUFFER_H
+
+#include <stdlib.h>
```

Comment:
Is this to pull in `size_t`? You can pull in `stddef.h` to get it directly (I believe that `stdlib.h` is defined to pull in `stddef.h`, but I'm not 100% certain this is portable).

Reply:
Yep. Have updated :)